src/lib/util/flag-resolver.test.ts

```diff
@@ -14,9 +14,11 @@ test('should produce UI flags with extra dynamic flags', () => {
         ...defaultExperimentalOptions,
         flags: { extraFlag: false },
     };
+    // @ts-expect-error
```

Comment:
can we get rid of those?

Reply:
e.g.  const resolver = new FlagResolver(config as IExperimentalOptions);

Reply:
casting seems to be more idiomatic in TS tests than expecting errors

Reply:
Good point, addressed in https://github.com/Unleash/unleash/pull/2571/commits/e226e30584d98ff504660ed1c20d9ab206f855cf.